package.json

```diff
@@ -22,6 +22,7 @@
     "craco-less": "^1.17.1",
     "date-fns": "^2.21.1",
     "dotenv-cli": "^4.0.0",
+    "formik": "^2.2.8",
```

Comment:
This feels like a departure from using Ant design. I'm curious as to what the benefits of this package are?